common/gnome-shell/3.36/sass/_common.scss

```diff
@@ -2644,7 +2644,7 @@ StScrollBar {
 }
 
 #lockDialogGroup {
-  background-color: if($variant!='dark', darken($bg_color, 10%), darken($bg_color, 5%));
+  background-color: if($variant != 'lighter', opacify($osd_bg_color, 1), opacify(darken($osd_bg_color, 8%), 1));
```

Comment:
I'm assuming the reason for `darken()`ing the background for lighter variant, is to avoid the blinding near-full white background. However, maybe we could use some pre-existing color, e.g. `$header_bg` or `$panel_bg` for better coherency?

Reply:
Intent is indeed to avoid glaring light background - and none of the other color variables is "dark" enough

Reply:
The problem is that the login screen buttons etc. must have the corresponding background color. I did test this with some of the pre-existing color variables, but even they looked off. Please stick with the undarkened `$osd_bg_color` for all variants. If that's too bright, users can just opt to use the other variants.

Reply:
Ok, done